pennylane/operation.py

```diff
@@ -695,6 +695,25 @@ def hash(self):
             )
         )
 
+    def __eq__(self, other):
+        warnings.warn(
+            "The behaviour of Operator.__eq__ will be updated soon. Currently, op1 == op2 is "
```

Comment:
```suggestion
            "The behaviour of Operator equality will be updated soon. Currently, op1 == op2 is "
```
(for users unfamiliar with dunder methods)